tests/app_manager.rs

```diff
@@ -145,3 +145,96 @@ async fn test_manager_create_spend() {
+                        address: Address::from_str(
+                            "tb1qkldgvljmjpxrjq2ev5qxe8dvhn0dph9q85pwtfkjeanmwdue2akqj4twxj"
+                        )
+                        .unwrap(),
+                        amount: Amount::from_sat(500),
+                        derivation_index: bip32::ChildNumber::from_normal_idx(0).unwrap(),
+                        status: VaultStatus::Active,
+                        txid: bitcoin::Txid::from_str(
+                            "a1075db55d416d3ca199f55b6084e2115b9345e16c5cf302fc80e9d5fbf5d48d"
+                        )
+                        .unwrap(),
+                        vout: 0,
+                        blockheight: Some(1),
+                        delegated_at: None,
+                        secured_at: Some(1),
+                        funded_at: Some(1),
+                        moved_at: None
+                    },
+                    Vault {
+                        address: Address::from_str(
+                            "tb1qkldgvljmjpxrjq2ev5qxe8dvhn0dph9q85pwtfkjeanmwdue2akqj4twxj"
+                        )
+                        .unwrap(),
+                        amount: Amount::from_sat(700),
+                        derivation_index: bip32::ChildNumber::from_normal_idx(0).unwrap(),
+                        status: VaultStatus::Active,
+                        txid: bitcoin::Txid::from_str(
+                            "a1075db55d416d3ca199f55b6084e2115b9345e16c5cf302fc80e9d5fbf5d48d"
+                        )
+                        .unwrap(),
+                        vout: 1,
+                        blockheight: Some(1),
+                        delegated_at: None,
+                        secured_at: Some(1),
+                        funded_at: Some(1),
+                        moved_at: None
+                    },
+                ]
+            })),
+        ),
+        (
+            Some(json!({"method": "setspendtx", "params":
+                        Some(
+                            "a1075db55d416d3ca199f55b6084e2115b9345e16c5cf302fc80e9d5fbf5d48d")
+            })),
+            Ok(json!({})),
+        ),
+    ]);
+
+    let sandbox: Sandbox<ManagerImportSendTransactionState> =
+        Sandbox::new(ManagerImportSendTransactionState::new());
+
+    let client = daemon.run();
+    let ctx = Context::new(
+        ConfigContext {
+            daemon: random_daemon_config(),
+            gui: GUIConfig::new(PathBuf::from_str("revault_gui.toml").unwrap()),
+        },
+        Arc::new(RevaultD::new(client)),
+        Converter::new(bitcoin::Network::Bitcoin),
+        Role::Stakeholder,
+        Menu::DelegateFunds,
+        Box::new(|| Box::pin(no_hardware_wallet())),
+    );
+    let sandbox = sandbox
+        .update(
+            &ctx,
+            Message::SpendTx(SpendTxMessage::PsbtEdited(psbt_string)),
+        )
+        .await;
+
+    let sandbox = sandbox
+        .update(&ctx, Message::SpendTx(SpendTxMessage::Updated(Ok(()))))
```

Comment:
You should first trigger the import, this message will be the result of the command. 
Do `SpendTxMessage::Import` instead